### DISCUSSION TITLE
PPA for Ubuntu not yet updated with 4.6.x

### DISCUSSION THREAD

**Discussion by Oldbwl** [quodlibet/quodlibet#4366] [General]
Does anyone know when/if the PPA will contain the the latest version 4.6.0? I only ask as I want to do quite a bit of work on my collection and want to test the new version before I start.

Many thanks

**Comment by Moonbase59** [comment]
Yeah, I asked here: https://github.com/quodlibet/quodlibet/issues/4353 and was told 4.6 is "in progress" but nothing much has happened since. Would be nice if not only devs with building environments but also normal end users could get 4.6, or even a nightly or beta 4.7…

I do know it’s quite some effort for a _person_, but it might make much sense to set up a CI so a _machine_ can do all the tedious work of fabricating binaries for miscellaneous platforms.

**Comment by Oldbwl** [comment]
I think you will find if you go to the website and github, that 4.6.0 HAS
been released, in all respects except the PPA.

On Fri, 1 Sept 2023 at 18:14, Matthias C. Hormann ***@***.***>
wrote:

> Yeah, I asked here: #4353
> <https://github.com/quodlibet/quodlibet/issues/4353> and was told 4.6 is
> "in progress" but nothing much has happened since. Would be nice if not
> only devs with building environments but also normal endusers could get
> 4.6, or even a nightly or beta 4.7…
>
> I do know it’s quite some effort for a *person*, but it might make much
> sense to set up a CI so a *machine* can do all the tedious work of
> fabricating binaries for miscellaneous platforms.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quodlibet/quodlibet/discussions/4366#discussioncomment-6888041>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABUNSCMW4KSYXBDCUS7POZ3XYIJYTANCNFSM6AAAAAA4GPNCME>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Dan - Growing old disgracefully

**Comment by Moonbase59** [comment]
Hmm. I only see some ".exe" stuff there (probably Windows), "tar.gz" (which, when unpacked, look like a bunch of source code), and the Source Code again.

That’s nice, but nothing for Debian/Ubuntu/Mint (`.deb`), no RPMs, no PKBUILD. How would an Arch, Debian-derivative or other Linux end user install that? I actually _have_ the lazka PPA installed on my Mint (love the Linux "centralized updates" idea), but the last builds there are 74/125 weeks old! While the docs say to use that…

Seems end users don’t get much love here. A pity, since QL does many things _right_.

**Comment by Oldbwl** [comment]
PPA now updated, thank you

**Comment by Moonbase59** [comment]
Yup, indeed. Thanks from me, too!
